docs/source/running.rst

```diff
@@ -0,0 +1,61 @@
+.. _running:
+
```

Comment:
Podle mě by chtělo připsat, ještě, jak se to pouští bez SGE, aby byl tenhle dokument úplnej.